exporter/clickhousetracesexporter/migrations/000024_span_attributes_keys.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS signoz_traces.span_attributes_keys ON CLUSTER cluster (
+    tagKey LowCardinality(String) CODEC(ZSTD(1)),
+    tagType Enum('tag', 'resource') CODEC(ZSTD(1)),
+    dataType Enum('string', 'bool', 'float64') CODEC(ZSTD(1)),
+    isColumn bool CODEC(ZSTD(1)),
+) ENGINE ReplacingMergeTree
+ORDER BY (tagKey, tagType, dataType, isColumn);
```

Comment:
Does this need to keep the values forever? Can it have the same maximum TTL as the index table?

Reply:
But it's expected that keys won't be more than 10s of thousands, keys table of logs also doesn't have a TTL